docs/en/latest/getting-started.md

```diff
@@ -177,15 +198,13 @@ This response indicates that Apache APISIX is running successfully.
 }
 ```
 
-## Step 2: Create a Route
-
-[Routes](./architecture-design/route.md) matches the client's requests based on defined rules, loads and executes the corresponding plugins, and forwards the request to the specified upstream.
+## Create a Route
 
-From the previous step, we have a running instance of Apache APISIX in Docker. Now let's create a Route.
+From the previous step, we have a running instance of APISIX in Docker. Now let's create a Route.
 
-Apache APISIX provides a powerful [Admin API](./admin-api.md) and [APISIX Dashboard](https://github.com/apache/apisix-dashboard). Here, we will use the Admin API to create a Route and connect it to an [Upstream](./architecture-design/upstream.md) service. When a request arrives, Apache APISIX will forward the request to the specified Upstream service.
+APISIX provides a powerful [Admin API](./admin-api.md) and [APISIX Dashboard](https://github.com/apache/apisix-dashboard). Here, we will use the Admin API to create a Route and connect it to an [Upstream](./architecture-design/upstream.md) service. When a request arrives, APISIX will forward the request to the specified Upstream service.
 
-We will create a sample configuration for our Route object so that Apache APISIX can forward the request to the corresponding Upstream service.
+We will configure the Route so that APISIX can forward the request to the corresponding Upstream service:
```

Comment:
When I use httpbin.org as upstream, it returns the error code `503`. I see two Nginx containers already running. Maybe we can change the upstream to `127.0.0.1:9081`.
![image](https://user-images.githubusercontent.com/97138894/165259805-26d92687-ccac-4c2c-8545-23564fadc5ae.png)

![image](https://user-images.githubusercontent.com/97138894/165259632-bb15fe98-3066-4620-a0ba-18c804afca0a.png)

![image](https://user-images.githubusercontent.com/97138894/165258840-21d12626-c79b-424f-a116-57ce4a37584b.png)

Reply:
That is our dataplane right?

Reply:
Sorry, I didn't understand what you meant. I deployed Apache APISIX according to the documentation and I got an error when testing the routes.

Reply:
:9081 and :9082 is for the Nginx service right? APISIX is on :9080. Your error could be from HTTPbin. They might have been down for some reason.

It is working fine for me now:

![Screenshot 2022-04-26 at 4 45 05 PM](https://user-images.githubusercontent.com/49474499/165288510-d69fa491-1c0d-4cef-b27f-2b7a59e96f88.png)

Reply:
yes. `:9081` and `:9082` are for the Nginx service. My error is from HTTPbin. So if we use the local Nginx service, this error may not occur.

Reply:
Here, HTTPbin acts as the API the user is trying to expose with APISIX. It is a sample application of sorts. If HTTPbin is unavailable most of the time, we can maybe have it running locally but for me, HTTPbin was working on most cases. // @juzhiyuan @spacewander

Reply:
Yes, and let's use HTTPbin as upstream service here :)